nest-forms-backend/src/nases/utils-services/tokens.nases.service.ts

```diff
@@ -517,8 +517,16 @@ export default class NasesUtilsService {
         },
       }
     }
+
+    // skip saving the message to outbox if we are sending it to ourselves as it will be available in inbox
+    const sktalkReceiveFn =
```

Comment:
My preference for verbosity comes again, I strongly suggest to add let's say `method` argument to `sendMessageNases`, which will be like:

```
 async sendMessageNases(
    jwtToken: string,
    data: Forms,
    method: 'receivePost' | 'receiveAndSaveToOutboxPost' // feel free to use enum
    senderUri?: string,
  )
```

Missing `senderUri` = we want to skip outbox is just assumption that is true right now, but might change in the future.

Reply:
However the whole implementation is not optimal, I don't like this part:
```
    const senderId =
      senderUri ?? this.configService.get<string>('NASES_SENDER_URI') ?? ''
```  
In the future, I would refactor it to have two modes:
 - sending to ourselves (senderUri cannot be passed) and is provided from env variables when creating message
 - eID send (senderUri must be passed)

Current implementation just assumes when senderUri is missing we send from ourselves.

Reply:
I agree with this approach. For now we need the functionality urgently, and we will implement this behavior as soon as feasible and practicable.